test/private/execute_hash_stage.m

```diff
@@ -28,7 +28,7 @@
     % By storing the hash table in a persistent variable, the amount of disk
     % operations is minimized (i.e. reading the file, parsing it and storing its
     % data in a MATLAB struct), as this is only done once a new test suite is
-    % executed. To clear this persistent storage, run |clear functions|.
+    % executed. To clear this persistent storage, run `clear getReferenceHash`.
```

Comment:
See http://nl.mathworks.com/help/matlab/matlab_prog/marking-up-matlab-comments-for-publishing.html for MATLAB Markdown. I'm not particularly fond of that notation either (I like GFM better in general), but it might be more convenient in the long run perhaps.